clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -864,7 +864,6 @@ public void handle(SyncGroupResponse syncResponse,
      * @return A request future which indicates the completion of the metadata request
      */
     private RequestFuture<Void> sendFindCoordinatorRequest(Node node) {
-        // initiate the group metadata request
```

Comment:
The javadoc above also mention the GroupMetadata request. Can we update that as well?

Reply:
Ah, thanks. The node selection is also done by the caller, so I updated that as well.